packages/core-components/src/layout/Sidebar/SidebarLogOutButton.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import { SidebarItem } from '@backstage/core-components';
+import LockIcon from '@material-ui/icons/Lock';
+import { IconComponent } from '@backstage/core-plugin-api';
+
+/**
+ * Button for sidebar that logs out user
+ *
+ * @public
+ */
+export const SidebarLogOutButton = (props: {
```

Comment:
Not really something we want to add as a general pattern I would think, keeping this in settings instead. Could you move this to be implemented in `packages/dev-utils` instead? Or do you feel strongly about this being useful elsewhere

Reply:
Depends what you guys think. We have our own logout button in our instance in local env which is nice while developing and changing accounts. This can also be removed completely or moved to dev utils, what do you think?

Reply:
Thinking dev utils is the best place for it for now, just as part of the dev app, not public API. Then if that's a feature people miss in the main app during development we could consider adding it here or some other place.